web/src/reducers/budget.js

```diff
@@ -458,18 +458,19 @@ const buildBudget = incomingBigState => {
       const quarterlyFFP = newState.federalShareByFFYQuarter[ffpSource];
 
       ['contractors', 'expenses', 'statePersonnel'].forEach(prop => {
-        const propCostType = prop === 'contractors' ? 'contractors' : 'state';
+        const ffpType = prop === 'contractors' ? 'contractors' : 'state';
```

Comment:
Does `state` in this instance refer to something different than the `state` that `inHouse` replaced?

Reply:
Kind of yes, kind of now. It's the same type of information, but this one comes from the user input rather than the budget calculations. There's #2066 to finish out the change, but the two seemed big enough to be separate PRs.